modules/ecs-service/versions.tf

```
@@ -3,8 +3,16 @@ terraform {
 
   required_providers {
     aws = {
-      source  = "hashicorp/aws"
-      version = ">= 4.0"
+      source = "hashicorp/aws"
+      ## 4.66.0 version cause error
+      ## │ Error: listing tags for Application Auto Scaling Target (): InvalidParameter: 1 validation error(s) found.
+      ## │ - minimum field size of 1, ListTagsForResourceInput.ResourceARN.
+      ## │
+      ## │
+      ## │   with module.ecs_cloudwatch_autoscaling[0].aws_appautoscaling_target.default[0],
+      ## │   on .terraform/modules/ecs_cloudwatch_autoscaling/main.tf line 15, in resource "aws_appautoscaling_target" "default":
+      ## │   15: resource "aws_appautoscaling_target" "default" {
+      version = ">= 4.0.0, <= 4.65.0"
```

They fixed it in 4.66.1

Id change this to be greater than or equal to 4.66.1

```suggestion
      version = ">= 4.66.1"
```